org-configurations/default-configuration.yml

```diff
@@ -181,6 +181,10 @@ client:
         username:
           type: "email"
           pattern: '.+@.+\..+'
+          label:
+            en: "email"
+          placeholder:
+            en: "enter email"
```

Comment:
same here

Reply:
I added it here because if the user wants a custom label then he can add it for the username field.